frontend/src/features/projects/list/ProjectListView.tsx

```diff
@@ -130,15 +130,18 @@ export const ProjectListView: React.FC<IProps> = ({
       (value as any).agencies
         ? setFilter({ ...value, agencies: (value as any)?.agencies })
         : setFilter({ ...value });
-      if ((value as any).statusId) {
-        setFilter({ ...value, statusId: (value as any).statusId?.map((x: any) => x) });
-      } else {
+      if ((value as any).statusId.length === 0) {
         setFilter({ ...value });
-        setClearSelected(!clearSelected);
```

Comment:
Removed setClearSelected as this would cause an error: `Warning: Failed prop type: Invalid prop `selected` supplied to `TypeaheadContainer(WrappedTypeahead)`